package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -1002,32 +966,52 @@ export const MessageInputProvider = <
   }) => {
     const id = generateRandomId();
     const mimeType = lookup(file.name);
+
+    const blockedFile = blockedFiles?.some((x: string) => file.name?.includes(x));
```

Comment:
@Enigma-I-am what is the idea behind the `x` naming here?

Reply:
x is the file extension name in the list

Reply:
I'll give it a more descriptive name

Reply:
Something like

`const fileExtensionIsBlocked = blockedFileExtensions?.some((fileExtension: string) => file.name?.includes(fileExtension))` for example would make it easier to see what's actually going on

Reply:
✅